packages/common/src/types.ts

```diff
@@ -42,10 +42,11 @@ export interface UniversalWeb3ProviderInterface {
   chains?: Chain[];
   currentChain?: Chain;
 
+  // connect and return conneted accounts
   requestAccounts?: (wallet?: string) => Promise<Account[]>;
   disconnect?: () => Promise<void>;
+  switchChain?: (chainId: number) => Promise<void>;
```

Comment:
```suggestion
  switchChain?: (chain: Chain) => Promise<void>;
```
是不是可以直接把当前选中的链对象也返回，这样之后就不需要再去根据 id 查找了， Chain 当中应该也包含 id 的吧